### PR TITLE
Js fix console errs dynamic env params

### PR DESCRIPTION
**Pull request by jgstorey**
Renamed branch... removed & in name.

### CODE REVIEW

File: frontend/vite.config.js [hunk 1]
```diff
@@ -19,5 +19,16 @@ export default defineConfig({
...
+
+
+
+
+
+
+
+
+)
+
```
Comment (weskubo-cgi) [inline frontend/vite.config.js:34]: I'm not sure why this change is here. Seems to be mostly formatting and empty lines.

File: frontend/src/router/index.js [hunk 2]
```diff
@@ -170,7 +170,7 @@ const router = createRouter({
 router.beforeEach((to, _from, next) => {
   // this section is to set page title in vue store
   const appStore = useAppStore()
-  if (to && to.meta) {
+  if (to && to.meta.pageTitle) {
```
Comment (weskubo-cgi) [inline frontend/src/router/index.js:173]: I think it might make more sense to leave as it but ensure that Login route has a pageTitle defined.

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "msedge",
+            "request": "launch",
+            "name": "Launch Edge against localhost",
+            "url": "http://localhost:8082",
+            "webRoot": "${workspaceFolder}/frontend/src/*"
+        }
+    ]
+}
```
Comment (weskubo-cgi) [inline .vscode/launch.json:15]: I don't think we want this file in git. It's accounted for the in the .gitignore in backend and frontend but I think we will need a top level .gitignore as well. Just add the entry for this to start (you can copy from existing files).

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/router/index.js (hunk 2 above)
Comment (jgstorey) [reply frontend/src/router/index.js:173]: Note: adding page title to the check removes console warning... 
image

File: .vscode/launch.json (hunk 3 above)
Comment (jgstorey) [reply .vscode/launch.json:15]: Fixed.

File: frontend/vite.config.js (hunk 1 above)
Comment (jgstorey) [reply frontend/vite.config.js]: Fixed.

**Review by weskubo-cgi — APPROVED** [review]
